styles/components/_layout.scss

```diff
@@ -244,12 +281,30 @@
   margin-left: $pad;
 }
 
+@mixin colHGutter($colPad) {
```

Comment:
We do have a separate mixins file? Is the reasoning for putting this one here that it really would only be used here?